PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,15 @@
-- #TICKET_NUMBER
-
-### Reviewers
-**Functional:** 
-
-**Readability:** 
-
----
-
-
 ## Changes
 - add
 - remove
 - modify
 
-## QA steps (optional)
+## Reason for Change
```

Comment:
nit: could we put this section first? I find why more important than what.  And I want to be able to see the issue link right at the top as well.

Reply:
done